package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "react-html-element",
   "version": "0.0.1",
-  "description": "An extension of HTMLElement that allows for easy creation of Web Components using React.",
-  "main": "index.js",
+  "description": "An HTMLElement extension that provides a simple setup for Web Components using React.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
+    "test": "npm run build",
+    "build": "npm run lint && tsc",
     "lint": "eslint --ext .js,.jsx,.ts,.tsx --parser-options \"{project: './tsconfig.json'}\" ./src",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prepublishOnly": "npm run build"
```

Comment:
Curious what this `prepublish` task is for. Is this for the next step in getting autopublish ready?

Reply:
Yes. Just wanna make sure we're building every time we publish. I just added it because I was looking at the docs for that. I don't think it hurts to include it, even though we're not publishing yet.